docs/architecture.md

```diff
@@ -1,7 +1,7 @@
 ### Architecture
 
 Feather uses the Entity-Component-System architecture, also known as ECS. This architecture
-is widely used in the Rust gamedev ecosystem. 
+is widely used in the Rust game dev ecosystem. 
```

Comment:
Arguably `gamedev` could be considered right, but I think for the sake of complete correctness you could say `game development`